testdata/btf_map_init.c

```diff
@@ -0,0 +1,33 @@
+#include "common.h"
+
+#if __clang_major__ >= 9
+
+int __section("socket/1") tail_1() {
```

Comment:
is the "socket/1" required to be able to use it in PROG_ARRAY? libbpf doesn't support "/N" approach and I haven't planned to do so specifically because of the array initialization syntax you are implementing in this PR. So just trying to make sure this is not a requirement.

Reply:
Hey, thanks for the feedback! The numeric identifier in there might be a bit misleading, there is no `/N` magic going on there, the lib works exactly like libbpf. This prefix just specifies the prog type. Initially, I had 2 tail calls in this test, and I needed unique section names for them, since we don't yet implement multiple progs per section.

Your comment prompted me to investigate why some libbpf `section_defs` end in `/`, and turns out the part after the slash specifies the object to attach the program to. :slightly_smiling_face:  I'll just rename this to `"socket"`.